draft-ietf-ppm-dap.md

```diff
@@ -1997,7 +2039,7 @@ struct {
 } PrepareResp;
 ~~~
 
-Next the Helper decrypts each of its remaining report shares as described in
```

Comment:
This is the first step of computing `AggregationJobResp`, not an intermediate step.